mmv1/products/discoveryengine/product.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Discoveryengine
```

Comment:
I believe this should be `DiscoveryEngine`

Reply:
Thanks. Renamed the product to `DiscoveryEngine` and updated the product name in corresponding test examples.